README.md

```diff
@@ -1,10 +1,10 @@
 # Pop CLI
 
-<img src="/.icons/logo.jpeg"></img>
+<img src="https://github.com/r0gue-io/pop-cli/blob/main/.icons/logo.jpeg"></img>
 
 An all-in-one tool for Polkadot development.
 
-## Install
+## Installation
```

Comment:
I checked cargo-contracts and subxt as examples, and they provide installation instructions from crates.io. So we should add it.
I personally like to show how to do install `pop-cli` it from GitHub as well.  Can we include an explanation of both options?